.github/pull_request_template.md

```diff
@@ -57,5 +57,10 @@ If there are user-facing changes then we may require documentation to be updated
 -->
 
 <!--
-If there are any breaking changes to public APIs, please add the `breaking-change` label.
--->
\ No newline at end of file
+If there are any breaking changes to public APIs, please check the box below:
+-->
+* [ ] This PR includes breaking changes to public APIs.
+<!--
+Check the box below if the changes fix either a security vulnerability or a bug that caused incorrect or invalid data to be produced. We use this to highlight fixes to issues that may affect users without their knowledge. For this reason, fixing bugs that cause errors or crashes don't count, since those are usually obvious.
+-->
+* [ ] This PR contains a "Critical Fix".
```

Comment:
Minor argument against using checkboxes for this: this adds a "todo" list to each PR, and then the GitHub UI will show this for example as "1 of 2 tasks" done in the pull request listing

(ideally github would also allow you use a "form" and not just a template, so this could be an actual checkbox that would add the label)

Reply:
Also, if there are breaking changes to the public API, we should maybe ask to actually list / describe the exact change(s), instead of just checking the box? (and potentially give some rationale for making the change, although that could also refer to the issue)

Reply:
> (ideally github would also allow you use a "form" and not just a template, so this could be an actual checkbox that would add the label)

Yeah I think to get close to that we could have our bot add the labels if these boxes are checked (or using some other indicator).

Reply:
Based on your feedback, I've reverted the instructions to "uncomment the line below" and also requested the creator provide explanation on which specific changes are breaking or critical fix.